Leno_beta1.5/src/Material.cpp

```diff
@@ -71,21 +86,21 @@ double Material::electronDensity(double phin) {
 		//TODO: add Nta options in the input file. tried but ExtractData will not run correctly
 		// for DRC2016 Nta = 1600
 		// Feb 6th, 2017. Change 1600 to Nta. (By Frank)
-		if (Nta-0.0<1e-16) {
+		if (Nta-0.0 < 1e-16) {
```

Comment:
I believe C++ double precision is 1e-16, so it is good enough to compare with 1e-14.